core/store/src/trie/mem/arena/mod.rs

```diff
@@ -261,6 +261,7 @@ impl<'a> ArenaSliceMut<'a> {
 mod tests {
     #[test]
     fn test_arena_mmap() {
+        println!("{:?}", rustix::process::getrlimit(rustix::process::Resource::Data));
```

Comment:
did you mean to leave that in?

Reply:
Yes, currently debugging https://github.com/near/nearcore/issues/9634. Will remove once I have sufficient details.